Core/MIPS/MIPS.cpp

```diff
@@ -308,8 +306,6 @@ void MIPSState::DoState(PointerWrap &p) {
 		Do(p, fcr0_unused);
 	}
 	Do(p, fcr31);
-	Do(p, rng.m_w);
-	Do(p, rng.m_z);
```

Comment:
Here, you need to check if the savestate version is 3 or below, in which case you still do the old steps, for example to local dummy variables. There are many examples around the code for this.

In this case it would simply be, I think:

```
if (s <= 2) {
   uint32_t dummy;
   Do(p, dummy);
   Do(p, dummy);
}

With that, old savestates should load fine (though new savestates won't load in old versions, but that's fine.)

}
```

Reply:
You mean `if (s <= 3)`, not `if (s <= 2)`?

Reply:
Oh sorry yes, you are increasing from 3 to 4, not from 2 to 3.